vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -233,7 +233,7 @@ enum Result {
   AC_TABLE_EMPTY = 14,
   NOT_ENOUGH_FUNDS = 15,
   TIP_ERROR = 16,
-  CORRUPTED_WALLET = 17,
+  CORRUPTED_DATA = 17,
```

Comment:
I changed this to more generic result so that we can reuse it

Reply:
Would it not be better to differentiate between different types of data corruption to help support?

Reply:
we can easily see which flow return this error, so support will know which is which. If you get CORRUPTED_DATA from loading JSON or reading wallet you know what is the problem, compared if you get this from promotion flow